docs/source/converting.rst

```diff
@@ -54,9 +54,9 @@ Lightning automates most of the trining for you, the epoch and batch iterations,
 ===========================
 To add an (optional) validation loop add logic to :func:`pytorch_lightning.core.LightningModule.validation_step` hook (make sure to use the hook parameters, self in this case).
 
-.. testcode::
+.. code-block::
```

Comment:
what was wrong with this test?

Reply:
it was failing on master since pl is not defined here. Also all other blocks are code-blocks up there.

Reply:
You can use it without pl...
Cc: @awaelchli

Reply:
ok, fixed